ServiceTier/Services/TaskService.cs

```diff
@@ -223,6 +223,58 @@ public IEnumerable<TaskDto> ImportTasksFromXml(IFormFile uploadFile, int userId)
             return AddTaskToDatabase(listOfTasksDTO, userId);
         }
 
+        public IEnumerable<TaskRecordDto> GetAllRecordsByUserId(int userId)
+        {
+
+            var listOfTasks = this.UnitOfWork.Tasks.GetAllEntitiesByFilter(task => task.UserId == userId);
+            var listOfTaskRecords = (from tasks in listOfTasks
```

Comment:
inconsistency. you should either use query syntax or method syntax, but not both at the same time in one file (I'd prefer method synt)